src/labone/node_info.py

```diff
@@ -72,11 +72,44 @@ class NodeInfo:
     def __init__(self, info: NodeInfoType):
         self._info: NodeInfoType = info
 
+    @classmethod
+    def plain_default_info(cls, path: str | None = None) -> NodeInfoType:
+        """Get default info for a node.
+
+        Returns:
+            Default info for a node.
+        """
+        info: NodeInfoType = {
+            "Description": "",
+            "Properties": "Read, Write, Setting",
+            "Unit": "None",
+        }  # type: ignore[typeddict-item]
+        if path is not None:
+            info["Node"] = path
+        return info
+
+    def _checked_dict_access(
+        self,
+        item: t.Literal["Node", "Description", "Properties", "Type", "Unit", "Options"],
+    ) -> LabOneNodePath | str | NodeType | dict[str, str]:
+        if item not in self._info and item in [
+            "Description",
+            "Node",
+            "Properties",
+            "Type",
+            "Unit",
+            "Options",
+        ]:
+            msg = f"NodeInfo is incomplete. As '{item}'\
+              is missing, not all behavior is available."
+            raise KeyError(msg)
+        return self._info[item]
```

Comment:
issue: This does not look right. Either you typehint the item to be a string or you do test if its part of the literal list ... 
I would suggest you simply typehint a str